hr_holidays_meeting_name/models/hr_holidays.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
@adrienpeiffer my bad: there was some discussions but finally no guidelines.
Sorry about the noise